src/markdoc/layouts/Policy.svelte

```diff
@@ -22,28 +22,38 @@
 
     import { Main } from '$lib/layouts';
 
-    import { DEFAULT_DESCRIPTION, DEFAULT_HOST } from '$lib/utils/metadata';
+    import { DEFAULT_HOST } from '$lib/utils/metadata';
 
     import { TITLE_SUFFIX } from '$routes/titles';
 
     import TocNav from '$lib/components/TocNav.svelte';
     import TocRoot from '$lib/components/TocRoot.svelte';
     import { isHeaderHidden } from '$lib/layouts/Main.svelte';
     import { getContext, hasContext, setContext } from 'svelte';
+    import { page } from '$app/stores';
 
     export let title: string;
     export let transparentTableCells = false;
 
-    const seo = {
+    let seo = {
         title: title + TITLE_SUFFIX,
-        description: DEFAULT_DESCRIPTION,
+        description: 'Understand how we collect, use, and protect your personal information.',
         ogImage: DEFAULT_HOST + '/images/open-graph/website.png',
 
         APP_NAME: 'Appwrite'
     };
 
     let showToc = false;
 
+    // handle multiple policies
+    const pageId = $page.route.id;
+
+    if (pageId === '/cookies') {
+        seo.description = 'Read our cookie policy to understand your choices and manage your preferences.'
+    } else if (pageId === '/terms') {
+        seo.description = 'Review our Terms of Service to understand the rules and guidelines for using our open-source backend-as-a-service platform.'
+    }
```

Comment:
not sure I like this, probably we need to have a better foundation to allow tying the description in the markdoc pages.

lets add an `export let description: string;` and add those descriptions in the `.markdoc` files 👍🏻

Good find that we didnt have this available in policy pages.

Reply:
Makes sense. Will do!